dist/common/scripts/node_exporter_install

```diff
@@ -42,6 +42,11 @@ if __name__ == '__main__':
     if node_exporter_p.exists() or (bindir_p() / 'prometheus-node_exporter').exists():
         if force:
             print('node_exporter already installed, reinstalling')
+            try:
+                node_exporter = systemd_unit('node-exporter.service')
+                node_exporter.stop()
+            except:
+                pass
```

Comment:
If the service fails to stop, the error will not be caught

Reply:
Indeed a better method is to rename the file into the new position, and issue a restart.